### ISSUE TITLE
Copied content containing superscripts doesn't render correctly in the editor

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/gutenberg-mobile#2090]
**Describe the bug**

After copying and pasting content from a Wikipedia page into a post, I found that the superscript values displayed as `[SUP]` in the editor instead of showing the actual content.

**To Reproduce**

1. Go to a Wikipedia page such as the [Tiny house movement](https://en.wikipedia.org/wiki/Tiny_house_movement).
1. Copy any portion of the content containing superscripts.
1. In the app, go to My Sites > Posts > +.
1. Add a title and paste the content into the content area.
1. Preview the post.

**Expected behavior**

I expected that content with superscripts would display normally in the editor.

**Screenshots**

<img src="https://user-images.githubusercontent.com/1119271/78087368-198d8500-737e-11ea-8908-ef84dab85d73.PNG" width="200" alt="IMG_3647"> <img src="https://user-images.githubusercontent.com/1119271/78087367-18f4ee80-737e-11ea-8c70-1c2e746ae187.PNG" width="200" alt="IMG_3648"> <img src="https://user-images.githubusercontent.com/1119271/78087360-15616780-737e-11ea-8b12-609268cc6f1e.PNG" width="200" alt="IMG_3649"> 

**Smartphone (please complete the following information):**
 - Device: iPhone 11
 - OS: iOS 13.4
 - Version: WPiOS 14.5.0.1 TestFlight beta

**Comment by etoledom** [comment]
This looks like a tag that it is not supported by Aztec.
To me it seems more like a feature request in this case 🤔 

There's also a tinny difference in the HTML generated compared with Web:

**On web:**
```html
<sup>
    <a href="https://en.wikipedia.org/wiki/Tiny_house_movement#cite_note-1">
        [1]
    </a>
</sup>
```

**On iOS and Android:**
```html
<sup class="reference">
    <a href="https://en.wikipedia.org/wiki/Tiny_house_movement#cite_note-1">
        [1]
    </a>
</sup>
```
But Android is rendering it as expected.

I would ask @hypest about the priority of this.
cc @SergioEstevao

**Comment by designsimply** [comment]
Thanks! +1 for getting input on priority—I would vote medium or low unless it's a quick thing to fix.

Aside: I noticed that the same problem doesn't happen in the editor on Android.

<img src="https://user-images.githubusercontent.com/1119271/78285774-eeba4280-74dd-11ea-915f-6a4aa854a10c.png" width="200" alt="Screenshot_20200402-122914">
<sup>Tested with WPAndroid 14.5-rc-2 on Pixel 3 Android 10.</sup>

**Comment by hypest** [comment]
It does look like a medium-prio feature request, especially since based on what @etoledom mentioned the issue lies with Aztec (iOS) itself not currently supporting rendering. When Aztec-iOS supports it, it should be readily available in the block editor too. Can you move this ticket to the Aztec-iOS repo @designsimply ? Thanks!

In the meantime, if you want to tackle this @etoledom , feel free to 👍.

**Comment by SergioEstevao** [comment]
The sup and sub tags are now supported in Aztec iOS  and by consequence they are also supported in Gutenberg mobile. Available since version 1.28
